control/src/spi_interface.c

```diff
@@ -0,0 +1,90 @@
+
```

Comment:
Add file header to your sources like I have for the other source files

Reply:
Done